common/sensor/sensor.c

```diff
@@ -240,7 +242,12 @@ void sensor_poll_handler(void *arug0, void *arug1, void *arug2)
 
 	while (1) {
 		for (poll_num = 0; poll_num < SENSOR_NUM_MAX; poll_num++) {
-			if (sensor_poll_enable_flag == 0) { /* skip if disable sensor poll */
+			if (sensor_poll_enable_flag == false) { /* skip if disable sensor poll */
+
+				// Due to except skip sensor polling
+				if (is_sensor_ready_flag == false) {
+					is_sensor_ready_flag = true;
+				}
```

Comment:
If this is false, it sets it to true.
If this is already true, then it's true.

This can probably just be replaced with:
`s_sensor_ready_flag = true;`